src/utils/image-processor.js

```diff
@@ -60,7 +63,7 @@ const shrinkFile = function(file, settings) {
             canvas.width = canvas.height = 1
             df.notify(0.9)
             // console.log('to blob: ' + (new Date() - start))
-            if (exif) {
+            if (exif && !isExifApplied) {
```

Comment:
This will completely strip EXIF from images when `shrinkFile` if used. What we really need is alter `exif` orientation to 1 if `isExifApplied`. Look at `parseExifOrientation` how to do that.

Reply:
o no

Reply:
@homm I did it